.github/workflows/build_docker.yml

```
@@ -14,22 +14,34 @@ jobs:
         uses: actions/checkout@v2
 
```

```suggestion
      - name: Set SHORT_SHA environment variable
        run: echo "::set-env name=SHORT_SHA::$(git rev-parse --short "$GITHUB_SHA")"
```

If you want to simplify the workflow a bit, you can set the env var for this ahead of time and it will be available to subsequent steps

---

Yep I'm aware. Thanks. Using this in upcoming CD extension.